src/VNCVideoCapture.cpp

```diff
@@ -0,0 +1,206 @@
+#include <rfb/rfbclient.h>
+#include <signal.h>
+#include <time.h>
+#include "VNCVideoCapturer.h"
+
+#include "rtc_base/thread.h"
+#include "rtc_base/logging.h"
+#include "api/video_codecs/video_decoder.h"
+#include "media/base/videocapturer.h"
+#include "media/engine/internaldecoderfactory.h"
+#include "api/video/i420_buffer.h"
+#include "libyuv/video_common.h"
+#include "libyuv/convert.h"
+#include <iostream>
+
+char *host = "127.0.0.1";
+int port = 5900;
+
+static char * get_password (rfbClient *client) {
```

Comment:
@jbarber I dont think I can just pass this->onGetPassword  as it cant be passed as a function pointer (class functions need the context this). 

So this was a wrapper around it. Let me know how you would suggest to do this otherwise.